### DISCUSSION TITLE
Backward/Forward large_ff2 buttons CSS broken

### DISCUSSION THREAD

**Discussion by joefiesta** [Aris-t2/CustomCSSforFx#592] [General]
My backward/forward buttons changed.  Not exactly sure which version of FF caused this. FF currently 112.0.2.

Backward/forward buttons now appear as TWO PAIRS of backward/forward buttons; that is, each button is composed of a BACKWARD and a FORWARD icon.    They still function properly, however.

I am using CSS addition:

  

>  @import "./css/buttons/custom_backforward_large_ff2.css"; 

Removing this CSS entry fixed the problem, although there is now a large empty space between the two buttons and the URL bar.  

 Is "custom_backforward_large_ff2.css" broken?   

lmk if I need to submit my entire custom CSS.

**Comment by Aris-t2** [comment]
Make sure you always use the latest version of this projects files: https://github.com/Aris-t2/CustomCSSforFx/releases

The issue you are mentioning was fixed in v4.3.2.

**Reply by joefiesta** [reply]
Problem fixed!   thanks for the quick reply.    I had installed 4.3.5 but missed some changes.   My bad.
